tedi/components/form/slider/slider.component.html

```
@@ -0,0 +1,77 @@
+@if (label()) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)